detox/src/configuration/composeScriptsConfig.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @param {{
+ *  appsConfig: Record<string, Detox.DetoxAppConfig>
+ *  localConfig: Detox.DetoxConfiguration;
+ * }} options
+ * @returns {DetoxInternals.RuntimeScriptConfig[]}
+ */
+function composeScriptsConfig(options) {
```

Comment:
I think the term "commands" fits here better than "scripts" and is more aligned with other uses of it